From f357521de132e9d3290cd084edface86a50ae90e Mon Sep 17 00:00:00 2001
From: Rocky Hao <rocky.hao@rock-chips.com>
Date: Mon, 10 Apr 2017 18:33:34 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3368: add latency-bound property

add latency-bound property to ease user's config and debugging

Change-Id: I76fd2945e4fdecf251ea2f4b15bd6e90fd154145
Signed-off-by: Rocky Hao <rocky.hao@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3368.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3368.dtsi b/arch/arm64/boot/dts/rockchip/rk3368.dtsi
index 209ff0e1b792..28f3300cfed7 100644
--- a/arch/arm64/boot/dts/rockchip/rk3368.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3368.dtsi
@@ -747,6 +747,7 @@
 		nvmem-cell-names = "temp_adjust";
 		#thermal-sensor-cells = <1>;
 		hw-shut-temp = <95000>;
+		latency-bound = <50000>;
 		status = "disabled";
 	};
 
-- 
2.35.3

